extensions/3DTILES_tile_metadata/README.md

```diff
@@ -0,0 +1,161 @@
+# 3DTILES_tile_metadata
+
+## Overview
+
+This extension to 3D Tiles allows for storage of per-tile metadata in external binary buffers.
+
+## Indexing
+
+When using 3D Tiles 1.0, the tile metadata is indexed using the Depth First Traversal of the `tileset.json`.
+
+When using the `3DTILES_implicit_tiling` extension, the tile metadata is indexed using the metadata bitstream.
+
+## Properties Reference
+
+---------------------------------------
+### 3DTILES_tile_metadata Tileset JSON extension
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**properties**|`object`|An object containing one or more metadata properties associated with each tile.|No|
+|**bufferViews**|`array`|An array containing typed views into buffers|No|
+|**buffers**|`array`|An array of buffers.|No|
```

Comment:
For clarity there should be some sort of `count` property, and all properties should contain `count` number of values.

Reply:
Added `elementCount` property to `3DTILES_binary_buffers`.